Ardas-Legends-Development-Team/AL-frontend
# Fix/claimbuild detail model bahavior

fixes https://github.com/Ardas-Legends-Development-Team/AL-frontend/issues/68

testing:
- go to any faction detail modal
- go to claimbuild
- click on details
- the dropdowns will work as they should
- you dont open all dropdowns in the same row anymore
